### PR TITLE
fix(SystemsTable): RHICOMPL-3469 - Restore OS info for Name column

### PR DESCRIPTION
**Pull request by bastilian**
Before:

Screenshot 2022-12-08 at 11 35 34

After: 
Screenshot 2022-12-08 at 11 35 04

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: src/Utilities/helpers.js [hunk 1]
```diff
@@ -76,6 +76,8 @@ export const constructQuery = (columns) => {
   const query = gql`
     fragment NameColumn on System {
       name
+      osMajorVersion
+      osMinorVersion
```
[the remark this answers is not in the document]
Comment (bastilian) [reply src/Utilities/helpers.js:80]: @skateman We could do that, yeah, but it seems like a tiny optimisation, or? it doesn't have much affect on the query construction on the frontend.

**Comment by sthirugn** [comment]
thank you for the quick fix

**Review by adonispuente — APPROVED** [review]
Nice fix, LGTM!

**Pull request merged**